styles.css

```diff
@@ -0,0 +1,196 @@
+.header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 10px;
+}
+
+.union {
+  padding-right: 20px;
+}
+
+#logo {
+  text-decoration: none;
+  width: 106px;
+  height: 20px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 25px;
+  line-height: 20px;
+
+  /* identical to box height, or 111% */
+
+  display: flex;
+  align-items: center;
+  text-align: center;
+  padding-left: 17px;
+
+  /* Primary/B400 */
+
+  color: #6070ff;
+}
+
+.main-section {
+  background-image: url(/images/mobie.png);
+  position: absolute;
+  width: 375px;
+  height: 636.15px;
+  left: 0;
+  top: 75px;
+}
```

Comment:
- [ ] As stated in the project requirement In order to lay out the elements on the page you should use `flexbox` in all 2 sections, So Kindly use `flexbox` in the headline section.